src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax.model;
+
+public class FruitTransaction {
+    private OperationType operation;
+    private String fruit;
+    private int quantity;
+
+    public FruitTransaction(OperationType operation, String fruit, int quantity) {
```

Comment:
Follow the encapsulation principle. Consider making the constructor package-private if it's only used within the package, or private if it's used with a builder/factory method. It's a good practice to limit the visibility of your class elements.